source/stylesheets/_project.scss

```diff
@@ -647,6 +684,26 @@ $bp-tall-viewport           : 920px;
   padding: grid-height(1) 0 grid-height(2);
   position: relative;
 
+  &.neutral {
+    background: #fafafa;
+    margin-bottom: 0;
+    .heading-label {
+      color: #FF0D69;
+    }
+    .supportive {
+      font-family:"freight-text-pro", "Lucida Bright", "Droid Serif", Georgia, serif;
+    }
+    .brief {
+      color: $nearBlack;
+    }
+    .flow-item .flow-arrow {
+      border-top: 1px dashed $deepGray;
+      &:after {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3